### PR TITLE
fix(config): #CO-1654 prevent config multiplication

### PR DESCRIPTION
**Pull request by sjollois**
## Describe your changes
Prevent config multiplication
## Checklist tests
go to homework-assistance with admin rights and try to set config several times. For dev, check in mongo collection that there are only one config that remains
## Issue ticket number and link
[https://jira.support-ent.fr/browse/CO-1654](url)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java [hunk 1]
```diff
@@ -7,12 +7,15 @@ private Field() {
     }
 
     public static final String ID = "id";
+    public static final String _ID = "_id";
     public static final String USERDATA = "userdata";
     public static final String DESTINATION = "destination";
     public static final String ERROR = "error";
+    public static final String ok = "ok";
```
Comment (qpic-cgi) [inline src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: tu as déjà une const "OK" l.16

**Review by qpic-cgi — APPROVED** [review]

File: src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java (hunk 1 above)
Comment (sjollois) [reply src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: oui mais elle est en majuscules :(

Comment (qpic-cgi) [reply src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: tu as besoin d'elle en minuscule ? 🤔

Comment (sjollois) [reply src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: oui pour checker la réponse Event Bus

**Review by qperie — APPROVED** [review]

File: src/main/java/fr/openent/homeworkAssistance/service/impl/DefaultConfigurationService.java [hunk 2]
```diff
@@ -1,39 +1,96 @@
...
     public void get(Handler<Either<String, JsonObject>> handler) {
         mongo.findOne(collection, new JsonObject(), event -> {
-            JsonObject result = new JsonObject();
-            if(event.body().containsKey("result"))
-                result = event.body().getJsonObject("result");
-            if(result.containsKey("messages"))
-                handler.handle(new Either.Right(result));
-            else
-            handler.handle(new Either.Left<>("No waiting orders"));
+            if (event.body().getString(Field.STATUS).equals(Field.ok)) {
```
Comment (qpic-cgi) [inline src/main/java/fr/openent/homeworkAssistance/service/impl/DefaultConfigurationService.java:23]: Field.OK

**Review by qpic-cgi — CHANGES_REQUESTED** [review]

File: src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java (hunk 1 above)
Comment (sjollois) [reply src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: mais non c'est ODE qui renvoit "ok" et pas "OK" du coup à moins de mettre un upperCase() à chaque fois (qui est très moche), je fais ça

Comment (qpic-cgi) [reply src/main/java/fr/openent/homeworkAssistance/core/constants/Field.java:14]: Autant pour moi, j'étais persuadé que dans ton Field.java, le champ "OK" => "ok"
alors que non, c'est "OK" => "OK".
eh bien, c'est approuvé du coup !

**Review by JinDouang — APPROVED** [review]

**Pull request merged**